csharp/ql/lib/semmlecode.csharp.dbscheme.stats

```diff
@@ -948,6 +948,14 @@
             <k>@with_expr</k>
             <v>101</v>
         </e>
+        <e>
+            <k>@list_pattern_expr</k>
+            <v>0</v>
```

Comment:
AFAIK these should be > 0?

Reply:
May I ask why they should be > 0 if the databases that the stats are based on doesn't contain any uses?
It is my impression that If there are no uses then they will be `0` when the stats are generated.
There already exist a couple of cases in the C# stats for file this:
One example
```
        <e>
            <k>@par_expr</k>
            <v>0</v>
        </e>
```

Reply:
I can set them to `1` instead if that is better, but most likely they will be overwritten again by `0`, when generating the stats.

Reply:
I thought the unit test that failed requires > 0, but I might be misremembering.

Reply:
The test seems to accept `0`.

Reply:
I don't know much about the semantics of these values, but 0 does seem a valid value. A couple of lines above, there's already:
```
        <e>
            <k>@par_expr</k>
            <v>0</v>
        </e>
```

Reply:
Then I mis-remembered, my apologies.

Reply:
Looks like everything is fine now. Sorry about the trouble @jketema !